File: FrontEnd/warehouse/src/app/modules/admin/city/models/city.model.ts

export class CityModel {
  id: string;
  code: string;
  name: string;
  countryId: string;
  isEdit: boolean;
  isActive: boolean;
  rowVersion: any;

  constructor(isEdit: boolean) {
    this.id = '';
    this.code = '';
    this.name = '';
    this.countryId = '';
    this.isEdit = isEdit;
    this.isActive = true;
  }
}

File: BackEnd/WareHouse/Services/Admin.Service/Models/CityModel.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace Admin.Service.Models
{
    /// <summary>
    /// City model.
    /// </summary>
    public class CityModel
    {
        public string Id { get; set; }

        [Required]
        [MaxLength(20)]
        public string Code { get; set; }

        [Required]
        [MaxLength(200)]
        public string Name { get; set; }

        [Required]
        public string CountryId { get; set; }

        public string CountryName { get; set; }

        public bool IsActive { get; set; }

        public byte[] RowVersion { get; set; }

        public string CurrentUserId { get; set; }

        public bool IsEdit { get; set; } = false;
    }
}
